frontend/apps/reader/modules/readerfont.lua

```diff
@@ -71,24 +74,18 @@ end
 
 function ReaderFont:setupTouchZones()
     if Device:isTouchDevice() then
-        self.ui:registerTouchZones({
```

Comment:
It's not completely clear to me why you changed this? It probably doesn't matter but tagging @houqp just in case.